src/create_py_project/update.py

```
@@ -0,0 +1,417 @@
+    selected = [o for o in available if o.value in selected_values]
+
+    table = Table(show_header=False, box=None, padding=(0, 2))
+    table.add_column(style="bold")
+    table.add_column(style="dim")
+
+    for opt in selected:
+        try:
+            opt.apply(target)
+            table.add_row("[green]✓[/green]", opt.label)
+        except Exception as exc:
+            table.add_row("[red]✗[/red]", f"{opt.label} — {exc}")
```

Ruff (0.15.15)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)